js/quote-generator.js

```diff
@@ -0,0 +1,55 @@
+const randomBtn = document.querySelector("#random-btn");
+const allAuthorsBtn = document.querySelector("#all-authors-btn");
+const quote = document.querySelector(".quote");
+let quoteAuthor = document.querySelector(".quote-author");
+let quoteGenre = document.querySelector(".quote-genre");
+const allAuthorsContainer = document.querySelector(".all-authors-container");
+const gridContainer = document.querySelector(".grid-container");
+const topButtons = document.querySelector(".top-buttons");
+const footer = document.querySelector("footer");
+
+async function getRandomQuote() {
+  const promise = await fetch("https://api.quotable.io/random");
+  const result = await promise.json();
+  quote.textContent = result.content;
+  quoteAuthor.textContent = result.author;
+  quoteGenre.textContent = result.tags.join(", ");
+}
+
+randomBtn.addEventListener("click", getRandomQuote);
+getRandomQuote();
+
+async function getAllAuthors() {
+  const promise = await fetch(
+    `https://api.quotable.io/authors?sortBy=quoteCount&order=desc&limit=10`
```

Comment:
You can save your base URL of your quote api in a constant 

`const BASE_URL = `https://api.quotable.io` and then reuse it